RecoLocalTracker/SiPixelRecHits/src/PixelCPETemplateReco.cc

```diff
@@ -131,8 +134,10 @@ PixelCPETemplateReco::localPosition(DetParam const & theDetParam, ClusterParam &
       ID = theDetParam.detTemplateId;
       if(ID0!=ID) cout<<" different id"<< ID<<" "<<ID0<<endl;
```

Comment:
Don't use cout in code that will be eventually run multihreaded: either LogSometing, or protect with one of your switches for the local runs

Reply:
Done.